env.example

```diff
@@ -0,0 +1,4 @@
+export DEV_DATABASE_NAME="jarvis"
+export DEV_DATABASE_USER="postgres"
+export DEV_DATABASE_PASSWORD="123456"
+export SECRET_KEY="7pgozr2jn7zs_o%i8id6=rddie!*0f0qy3$oy$(8231i^4*@u3"
```

Comment:
You will not need to have the `export` string if these are put in a file. You only need it when running it as a command directly on the command line